doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -15,6 +15,8 @@
       deserializing lists with complex type information
 
     @subsection qore_091_bug_fixes Bug Fixes in Qore
+    - fixed a bug with charset in the RestClient module (issue 3328)
```

Comment:
please leave a relnote for a module fix in the same format as the other modules - look a little below for lines like:
```
   - <a href="../../modules/Util/html/index.html">Util</a> module fixes:
      - fixed a bug in \c get_exception_string() with Java exceptions
        (<a href="https://github.com/qorelanguage/qore/issues/3304">issue 3304</a>)
```

Reply:
and please put the module fix lines all together...